test/jdk/java/awt/dnd/DropActionChangeTest.java

```diff
@@ -145,8 +143,11 @@ public void drop(DropTargetDropEvent e) {
     public void start() {
         try {
             robot = new Robot();
+            robot.setAutoDelay(100);
```

Comment:
```suggestion
            robot.setAutoDelay(100);
            robot.waitForIdle();
            robot.delay(500);
```

We usually wait a bit after calling `setVisible()`